stock_app.py

```diff
@@ -101,10 +111,12 @@ def analyze_and_plot(self, symbol, data):
         plt.show()
 
     def plot_stock_data(self, symbol, data):
-        df = data.reset_index()[['Date', 'Close']]
-        df = df.rename(columns={'Date': 'ds', 'Close': 'y'})
+        df = data.reset_index()[["Date", "Close"]]
+        df = df.rename(columns={"Date": "ds", "Close": "y"})
 
-        plt.plot(df['ds'], df['y'], label=symbol.replace(" ", "_"))  # Use a suitable label format
+        plt.plot(
+            df["ds"], df["y"], label=symbol.replace(" ", "_")
+        )  # Use a suitable label format
```

Comment:
Don't wrap lines to fit a comment.
```suggestion
        # Use a suitable label format
        plt.plot(df["ds"], df["y"], label=symbol.replace(" ", "_"))
```